### ISSUE TITLE
Error while uploading document "NullPointerException"

### ISSUE THREAD

**Issue by hahmyg** [webanno/webanno#673]
When I tried to import my WebAnno TSV3 file, 
system returns "NullPointerException" error

Even this (attached) file is exported from the WebAnno

[webannotsv3.txt](https://github.com/webanno/webanno/files/1385326/webannotsv3.txt)

**Comment by reckart** [comment]
I don't see any obvious mistakes in the file at a glance. Did you export it from WebAnno or generate it externally?

**Comment by hahmyg** [comment]
@reckart Thank you for your kind reply. 
I exported it from WebAnno.

If you want to test yourself, kindly, please let me know your email address.
I would give you URL and permission to the service

**Comment by reckart** [comment]
There seems to be a problem with the number of columns in the file. According to the header, there should be 12 columns, but there are only 11. It seems the `SurfaceForm` column is declared in the header, but it does not appear in the body. It should be the 4th column, but looking at the file, it seems to me that the 4th column is actually the POS tag.

Could you please try removing the following line from the file before importing:

```
#T_SP=de.tudarmstadt.ukp.dkpro.core.api.segmentation.type.SurfaceForm|value
```

**Comment by reckart** [comment]
It is probably a bug in the WebannoTsv3Writer since that header line should not be generated if the corresponding column is not also generated - or vice versa.

@hahmyg did you actually create any `SurfaceForm` in the document?

**Comment by reckart** [comment]
I have implemented a TSV3 reader and writer components (#676) which solve various known problems. These can also read/write your data after the problematic header line has been removed. I assume that it is a bug in the current TSV3 writer that cause the extra header line to appear. However, since there are alternative implementations now, I'm not going to try to debug the old code in depth anymore. I hope knowing that you have to remove the line solves your immediate problem. If not, please let me know.
